events/overview/tickets.rst

```diff
@@ -0,0 +1,53 @@
+===============
+Selling Tickets
+===============
+
+Save time, automate processes, and give attendees options to choose from in order to have the best
+purchase and payment methods available.
```

Comment:
Sentence is hard to read again. 

If you automate process you save time, and the second part is weird. Rewrite it please. :)